tools/firely-server-import.rst

```diff
@@ -265,4 +284,9 @@ Release 1.0.0
 * First public release
 * Performance: optimized memory consumption (especially, when reading large `*.ndjson` files)
 * Feature: quantitative values can be automatically canonicalized to UCUM values (see --useUcum CLI option)
-* Multiple smaller fixes to improve reliability and performance of the tool
\ No newline at end of file
+* Multiple smaller fixes to improve reliability and performance of the tool
+
+
+.. |br| raw:: html
```

Comment:
And why do we need these changes? :)

Reply:
`|br|` is a trick to force to start a new line. I use it, for instance, in line 185 to make the `e.g. ...` start from a new line. 

I saw that we use the same trick in many places. For instance `\firely-docs-firely-server\administration_api.rst`